lib/ZwaveClient.js

```diff
@@ -1333,14 +1340,14 @@ ZwaveClient.prototype.connect = async function () {
     } catch (error) {
       // destroy diver instance when it fails
       this.driver.destroy().catch(err => {
-        debug('Error while destroing driver ' + err.message)
+        logger.error('Error while destroing driver ' + err.message)
       })
       driverError.call(this, error)
-      debug('Retry connection in 3 seconds...')
+      logger.warn('Retry connection in 3 seconds...')
       this.reconnectTimeout = setTimeout(this.connect.bind(this), 3000)
     }
   } else {
-    debug('Driver already connected to', this.cfg.port)
+    logger.info('Driver already connected to', this.cfg.port)
```

Comment:
```suggestion
    logger.info(`Driver already connected to ${this.cfg.port}`)
```

I would convert all this loggers that are using commas `,` to use the template ``